components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -694,8 +694,9 @@ enum TransactionType {
   ERC20Approve = 2,
   ERC721TransferFrom = 3,
   ERC721SafeTransferFrom = 4,
-  Other = 5,
-  SolanaSystemTransfer = 6,  // transfering SOL
+  ETHSwap = 5,
+  Other = 6,
+  SolanaSystemTransfer = 6,  // transferring SOL
```

Comment:
The reason we predefine the values is to prevent this from happening.
You can't just change the existing enum values because it is casted as int and store on disk.
Please add new values to the end and also don't duplicate the value

Reply:
Oops, thanks for pointing out. I've fixed it.